docker/compose.yml

```diff
@@ -139,9 +169,14 @@ services:
   # More info: https://hub.docker.com/r/ixdotai/smtp
   #mailserver:
   #  image: ixdotai/smtp:latest
+  #  networks:
+  #    - mbin_external_network
   #  environment:
   #    - SMARTHOST_ADDRESS=mail.mysmtp.com
   #    - SMARTHOST_PORT=587
   #    - SMARTHOST_USER=myuser
   #    - SMARTHOST_PASSWORD=secret
   #    - SMARTHOST_ALIASES=*.mysmtp.com
+
+networks:
+  mbin_external_network:
```

Comment:
I'm not super familiar with docker networks. Based on the docs, it was using the default one, now this one, is there any concern of being unable to communicate? I guess it still exposes the same ports, it was only if people had custom containers they also ran, they'll have to move them over to the network as well?

Reply:
it has no other negative effects, I also kept it an external network (there is also a internal option). So it's the same as the default bridge, but it's advised to have a dedicated network instead of sharing the default bridge interface. So the benefit would be that another docker container can't access your mbin docker containers. Which also makes it safer.